src/encoded/static/components/dataset.js

```diff
@@ -8,9 +8,9 @@ import * as globals from './globals';
 import { DbxrefList } from './dbxref';
 import { FetchedItems } from './fetched';
```

Comment:
Sometimes I start code reviews from the smaller files then progress to large files; because the smaller files gives me context related to the larger ones... I made some comments  in other files that may apply but I won't repeat them.